brainpad/courses/vb/variables_and_nested_statements.md

```diff
@@ -1 +1,426 @@
 # Variables and Nested Statements
+## Overview 
+Students will learn about variables, strings, while loops and nested statements.
+
+## Guidelines 
+* Prerequisites: None
+* Ages 12 and up
+* PC setup with Visual Studio 2017, TinyCLR OS and Nuget packages installed. 
+* Supplies: BrainPad and microUSB cable.
+
+## Arguments
+Arguments are values passed on to methods. For example, to set the frequency of the buzzer, a value is passed on to that method. The 3000 is the arguments.
+
+```
+BrainPad.Buzzer.Start(3000)
+```
+Arguments are also used when adding delays. Here, the 0.2 is the argument.
+```
+BrainPad.Wait.Seconds(0.2)
+```
+Variables can be constants, like the numbers used above. They can also be variables, which will be explained in the coming sections. Also, arguments can also be something completely different!
+
+```
+    Public Sub BrainPadSetup()
```

Comment:
If we're not going to show the class declaration, remove the first indentation level.